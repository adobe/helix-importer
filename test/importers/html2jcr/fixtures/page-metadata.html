
<!DOCTYPE html>
<html>
  <head>
    <title>Test Page & Content</title>
    <link rel="canonical" href="https://main--xwalk-example--buuhuu.hlx.page/">
    <meta name="description" content="Test description">
    <meta name="datetime" content="2024-04-23T15:53:00.000+02:00"/>
    <meta name="robots" content="index, follow"/>
    <meta name="keywords" content="foo, bar, lorem & ipsum"/>
    <meta name="topics" content="one, two, three"/>
    <meta name="og:image" content="/content/dam/asset.jpg"/>
    <meta name="authortitle" content="John Doe"/>
    <meta name="abstract" content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla eu purus id tellus consequat pharetra. Fusce blandit felis ut dui congue, cursus suscipit est hendrerit."/>
    <meta property="og:title" content="Test Page">
    <meta property="og:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta name="template" content="salescontent">
    <meta name="viewport" content="width=device-width, initial-scale=1">
  </head>
  <body>
  </body>
</html>
