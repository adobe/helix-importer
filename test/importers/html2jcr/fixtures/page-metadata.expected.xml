<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page" jcr:title="Test Page &amp; Content" jcr:description="Test description" datetime="2024-04-23T15:53:00.000+02:00" cq:robotsTags="[index,follow]" keywords="[foo,bar,lorem &amp; ipsum]" topics="[one,two,three]" authorTitle="John Doe" abstract="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla eu purus id tellus consequat pharetra. Fusce blandit felis ut dui congue, cursus suscipit est hendrerit." template="salescontent">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root"/>
        <image jcr:primaryType="nt:unstructured" fileReference="/content/dam/asset.jpg"/>
    </jcr:content>
</jcr:root>